lib/src/layer/attribution_layer.dart

```diff
@@ -66,3 +85,382 @@ class AttributionWidget extends StatelessWidget {
         ),
       );
 }
+
+/// Base class for attributions that render themselves as widgets
+///
+/// Extended/implemented by [TextSourceAttribution] & [LogoSourceAttribution].
+@internal
+abstract class SourceAttribution extends StatelessWidget {
+  final void Function()? _onTap;
+
+  const SourceAttribution._({
+    super.key,
+    void Function()? onTap,
```

Comment:
You can use `VoidCallback?` instead of `void Function()?`

Reply:
Fixed in db944a1.